ticketutil/bugzilla.py

```diff
@@ -231,6 +234,45 @@ def add_comment(self, comment, **kwargs):
             logging.error("Error adding comment to ticket")
             logging.error(e.args[0])
 
+    def add_attachment(self, data, file_name, summary, **kwargs):
+        """
+        :param data: The content of the attachment which is base64 encoded.
+        :param file_name: The "file name" that will be displayed in the UI for this attachment.
+        :param summary: A short string describing the attachment.
+        :return:
+        """
+        if not self.ticket_id:
+            logging.error("No ticket ID associated with ticket object. Set ticket ID with set_ticket_id(ticket_id)")
+            return
+
+        # Read the contents from the file path, guess the mimetypes and update the params.
+        f = open(data, "+rb")
+        f_name = f.read()
+        content_type = mimetypes.guess_type(data)[0]
+        if not content_type:
+            content_type = 'application/octet-stream'
+        data = base64.standard_b64encode(f_name).decode()
+        params = {"data": data,
+                  "is_patch": False,
+                  "summary": summary,
+                  "file_name": file_name,
+                  "content_type": content_type}
+        params.update(kwargs)
+
+        # Attempt to change status of ticket.
+        try:
+            headers = {"Content-Type": "application/json"}
+            r = self.s.post("{0}/{1}/attachment".format(self.rest_url, self.ticket_id), json=params, headers=headers)
+            r.raise_for_status()
+            if 'message' in r.json():
+                logging.error("Change status error message: {0}".format(r.json()['message']))
```

Comment:
Update the logging messages in the lines below to represent adding an attachment, instead of changing status.